### DISCUSSION TITLE
Usage example of show_files_filter?

### DISCUSSION THREAD

**Discussion by ttddee** [Barugon/egui_file#35] [Q&A] [answered]
Hi,  

this is a great addition to egui.

I am struggling trying to figure out how to provide file name filtering, are there any examples available?

Thanks!

**Comment by Barugon** [comment, marked as answer]
[This](https://github.com/Barugon/cota/blob/4f70d1ca458781cf740117261e25ef92827c522b/src/app.rs#L221) filter will show only files that have names starting with "SotAChatLog_" and the extension "txt" and [this](https://github.com/Barugon/cota/blob/4f70d1ca458781cf740117261e25ef92827c522b/src/app.rs#L261) filter will show only files with the extension "sota".

**Reply by ttddee** [reply]
Thank you, that works.
